contrib/opentimelineio_contrib/adapters/contrib_adapters.plugin_manifest.json

```diff
@@ -1,18 +1,5 @@
 {
     "OTIO_SCHEMA" : "PluginManifest.1",
-    "adapters": [
-        {
-            "OTIO_SCHEMA": "Adapter.1",
-            "name": "xges",
-            "filepath": "xges.py",
-            "suffixes": ["xges"]
-        }
-    ],
-    "schemadefs" : [
-        {
-            "OTIO_SCHEMA" : "SchemaDef.1",
-            "name" : "xges",
-            "filepath" : "xges.py"
-        }
-    ]
+    "adapters": [],
+    "schemadefs" : []
 }
```

Comment:
Wild to see this empty.  Nice work @apetrynet and @rosborne132!  After this PR is the next step  to remove the contrib area altogether?

Reply:
It'll require some changes to https://github.com/AcademySoftwareFoundation/OpenTimelineIO/blob/main/src/py-opentimelineio/opentimelineio/plugins/manifest.py#L214, but it should be pretty straight forward. Unless by system you mean ecosystem, in which case we'll have to test using all adapters installed in the same environment.

Reply:
I'll have to look a bit closer, but what I meant was making sure search paths and where external plugins register don't require the contrib folder. I think it should be fairly straight forward, yes.
We should create a separate issue for this.